tn_api_hotel.yml

```diff
@@ -239,89 +312,449 @@ components:
   schemas:
     HotelSearchResponse:
```

Comment:
Add the 'NextResultReference' that's returned in the response. This can be used to query more hotels with the same search (ie: scroll down and want to load more).